### PR TITLE
docs: translate JavaScript in JSX with Curly Braces

### LINKED ISSUE #425: JavaScript in JSX with Curly Braces
Page: https://id.react.dev/learn/javascript-in-jsx-with-curly-braces

Assignee: @naufaldi

### PR DESCRIPTION
**Pull request by naufaldi**
Closes #425 

## Description

Translate the JavaScript in JSX with Curly Braces  page.
Page URL: [JavaScript in JSX with Curly Braces ](https://id.react.dev/learn/javascript-in-jsx-with-curly-braces)

### CODE REVIEW

File: src/content/learn/javascript-in-jsx-with-curly-braces.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
 ---
-title: JavaScript in JSX with Curly Braces
+title: JavaScript di JSX menggunakan Curly Braces
```
Comment (resir014) [inline src/content/learn/javascript-in-jsx-with-curly-braces.md:2]:
```suggestion
title: JavaScript di JSX menggunakan Kurung Kurawal
```

I think we can translate this term, yes? @mhaidarhanif @zainfathoni

File: src/content/learn/javascript-in-jsx-with-curly-braces.md [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
 
 </Intro>
 
 <YouWillLearn>
 
-* How to pass strings with quotes
-* How to reference a JavaScript variable inside JSX with curly braces
-* How to call a JavaScript function inside JSX with curly braces
-* How to use a JavaScript object inside JSX with curly braces
+* Bagaimana cara untuk oper *strings* dengan tanda kutip
```
Comment (resir014) [inline src/content/learn/javascript-in-jsx-with-curly-braces.md:13]:
```suggestion
* Bagaimana cara mengoper *string* dengan tanda kutip
```

Always convert loan words to its singular term.

File: src/content/learn/javascript-in-jsx-with-curly-braces.md [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
+* Bagaimana cara mereferensikan variabel didalam JSX dengan kurung kurawal
+* Bagaimana cara memanggil fungsi Javascript didalam JSX dengan kurung kurawal
+* Bagaimana cara menggunakan objek Javascript didalam JSX dengan kurung kurawal
 
 </YouWillLearn>
 
-## Passing strings with quotes {/*passing-strings-with-quotes*/}
+## Mengoper *strings* dengan tanda kutip {/*passing-strings-with-quotes*/}
```
Comment (resir014) [inline src/content/learn/javascript-in-jsx-with-curly-braces.md:20]:
```suggestion
## Mengoper string dengan tanda kutip {/*passing-strings-with-quotes*/}
```

- Singular term for loanwords.
- Don't italicise loanwords in headings.

File: src/content/learn/javascript-in-jsx-with-curly-braces.md [hunk 4]
```diff
@@ -41,9 +41,10 @@ export default function Avatar() {
 
 </Sandpack>
 
-Here, `"https://i.imgur.com/7vQD0fPs.jpg"` and `"Gregorio Y. Zara"` are being passed as strings.
+Disini, `"https://i.imgur.com/7vQD0fPs.jpg"` dan `"Gregorio Y. Zara"` sedang dioper sebagai *strings*.
```
Comment (resir014) [inline src/content/learn/javascript-in-jsx-with-curly-braces.md:44]:
```suggestion
Disini, `"https://i.imgur.com/7vQD0fPs.jpg"` dan `"Gregorio Y. Zara"` sedang dioper sebagai *string*.
```

Idem.

File: src/content/learn/javascript-in-jsx-with-curly-braces.md [hunk 5]
```diff
@@ -111,18 +112,18 @@ export default function TodoList() {
 
 </Sandpack>
 
-### Where to use curly braces {/*where-to-use-curly-braces*/}
+### Dimana menggunakan kurung kurawal? {/*where-to-use-curly-braces*/}
 
-You can only use curly braces in two ways inside JSX:
+Anda hanya dapat menggunakan kurung keriting (curly braces) dalam dua cara di dalam JSX:
```
Comment (resir014) [inline src/content/learn/javascript-in-jsx-with-curly-braces.md:117]:
```suggestion
Anda hanya dapat menggunakan kurung kurawal (curly braces) dalam dua cara di dalam JSX:
```

Let's make it consistent with the rest, use "kurung kurawal"

File: src/content/learn/javascript-in-jsx-with-curly-braces.md [hunk 6]
```diff
@@ -223,31 +226,32 @@ const person = {
...
 
 </Recap>
 
 <Challenges>
 
-#### Fix the mistake {/*fix-the-mistake*/}
+#### Perbaiki Kesalahan {/*fix-the-mistake*/}
 
-This code crashes with an error saying `Objects are not valid as a React child`:
+Kode ini mengalami crash dengan pesan galat yang menyatakan `Objects are not valid as a React child`:
```
Comment (resir014) [inline src/content/learn/javascript-in-jsx-with-curly-braces.md:254]:
```suggestion
Kode ini mengalami *crash* dengan pesan galat yang menyatakan `Objects are not valid as a React child`:
```

Missing italics.

**Review by resir014 — CHANGES_REQUESTED** [review]
First pass of review done. Some changes needed @naufaldi:

File: src/content/learn/javascript-in-jsx-with-curly-braces.md (hunk 4 above)
Comment (mazipan) [reply src/content/learn/javascript-in-jsx-with-curly-braces.md:44]: Add Disini -> Di sini

**Comment by mhaidarhanif** [comment]
@naufaldi need to resolve the requests from @resir014

File: src/content/learn/javascript-in-jsx-with-curly-braces.md [hunk 7]
```diff
@@ -111,18 +112,18 @@ export default function TodoList() {
...
+2. **Sebagai atribut** yang segera mengikuti tanda dengan `=`: `src={avatar}` akan membaca variabel `avatar`, tetapi `src="{avatar}"` akan mengoper string `"{avatar}"`.
 
-## Using "double curlies": CSS and other objects in JSX {/*using-double-curlies-css-and-other-objects-in-jsx*/}
+## Menggunakan "kurung kurawal ganda": CSS dan objek lain di JSX {/*using-double-curlies-css-and-other-objects-in-jsx*/}
 
-In addition to strings, numbers, and other JavaScript expressions, you can even pass objects in JSX. Objects are also denoted with curly braces, like `{ name: "Hedy Lamarr", inventions: 5 }`. Therefore, to pass a JS object in JSX, you must wrap the object in another pair of curly braces: `person={{ name: "Hedy Lamarr", inventions: 5 }}`.
+Selain string, angka, dan ekspresi JavaScript lainnya, Anda bahkan dapat oper objek dalam JSX. Objek juga ditandai dengan kurung kurawal, seperti `{ name: "Hedy Lamarr", inventions: 5 }`. Oleh karena itu, untuk oper objek JavaScript di JSX, Anda harus membungkus objek tersebut dalam sepasang kurung kurawal lainnya: `person={{ name: "Hedy Lamarr", inventions: 5 }}`.
```
Comment (mazipan) [inline src/content/learn/javascript-in-jsx-with-curly-braces.md:124]:
```suggestion
Selain string, angka, dan ekspresi JavaScript lainnya, Anda bahkan dapat mengoper objek dalam JSX. Objek juga ditandai dengan kurung kurawal, seperti `{ name: "Hedy Lamarr", inventions: 5 }`. Oleh karena itu, untuk mengoper objek JavaScript di JSX, Anda harus membungkus objek tersebut dalam sepasang kurung kurawal lainnya: `person={{ name: "Hedy Lamarr", inventions: 5 }}`.
```

**Comment by resir014** [comment]
@naufaldi Hi, any updates? If there are no updates within the next 24 hours, the maintainers will take over this PR.

**Review by resir014 — APPROVED** [review]
All good. Thanks!

**Pull request merged**